docs/guide/object-balls.md

```diff
@@ -76,7 +76,7 @@ FifaSoccerBall {
 
 ### FifaSoccerBall Field Summary
 
-- `size`: official FIFA ball sizes ranging from 1 to 5.
+- `size`: Defines the ball size according to FIFA standards. This field accepts the following values: `1`, `2`, `3`, `4` and `5`.
```

Comment:
```suggestion
- `size`: Defines the ball size according to FIFA standards. This field accepts the following values: `1`, `2`, `3`, `4`, and `5`.
```

> an Oxford comma to separate the last item in a list of three or more things.

https://prowritingaid.com/grammar/1008093/Is-it-grammatically-correct-to-put-a-comma-before-and-#:~:text=It%20is%20grammatically%20correct%20to,it%20as%20an%20Oxford%20comma.

The Oxford comma is used in American English only:
https://getproofed.com/writing-tips/differences-american-british-punctuation/